draft-ietf-tls-tls13.md

```diff
@@ -1056,6 +1056,12 @@ will not process the same data twice for the same connection) and
 an attacker will not be able to make 0-RTT data appear to be
 1-RTT data (because it is protected with different keys.)
 
+These properties mean that 0-RTT data must be specially treated by applications. They SHOULD be required
+to opt-in to 0-RTT data transmission or recieving it: implementations SHOULD NOT preappend recieved 0-RTT data
+to 1-RTT data automatically,but provide special functions to examine 0-RTT data.
+
+Protocols on top of TLS 1.3 should carefully consider when 0-RTT can be used.
```

Comment:
This sentence is a bit superfluous down here. Might be better to put a comma at the end of the first new sentence up top and stick this sentence fragment there. e.g. "... treated by applications, and protocols on top of ..."